### PR TITLE
Fixed the "Convert sensors card to ionic component" 

### LINKED ISSUE #2: Convert sensors Card to Ionic component
* Choose a matching Ionic component or a combination of components
* Use CSS Modules

### PR DESCRIPTION
**Pull request by andrewtran30**
Made some changes in the phone and tablet SensorsTab.js. Imported some dependencies so that the Ionic can integrate into React. Rather than using <ion-card> </ion-card> for example, I used `<IonCard></IonCard>` which is the correct Ionic notation. I think the old gui looked a little better but that's just my opinion!

Resolves #2

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "bootstrap": "^4.3.1",
     "bootswatch": "^4.3.1",
     "bowser": "^2.4.0",
+    "ionic": "^5.2.1",
```
Comment (grgur) [inline package.json]: We probably don't need ionic CLI for this app. We already handle integration with `@ionic/react`

File: src/Tablet/Sensors/SensorsTab.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import {
 } from "@/hooks/useSmartThings";
 
 import { Row, Col, Card } from "react-bootstrap";
```
Comment (grgur) [inline src/Tablet/Sensors/SensorsTab.js]: Probably no longer used

File: src/Phone/Sensors/SensorsTab.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from "react";
 import useConfig from "@/hooks/useConfig";
 
 import MQTT from "@/lib/MQTT";
+import { IonCard, IonHeader, IonCardHeader, IonCardContent, IonBadge } from "@ionic/react";
 
 import { Badge, Card } from "react-bootstrap";
```
Comment (grgur) [inline src/Phone/Sensors/SensorsTab.js]: Probably no longer used

File: src/Phone/Sensors/SensorsTab.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from "react";
 import useConfig from "@/hooks/useConfig";
 
 import MQTT from "@/lib/MQTT";
+import { IonCard, IonHeader, IonCardHeader, IonCardContent, IonBadge } from "@ionic/react";
```
Comment (grgur) [inline src/Phone/Sensors/SensorsTab.js]: Also IonHeader is not used.

File: src/Tablet/Sensors/SensorsTab.js [hunk 5]
```diff
@@ -10,6 +10,7 @@ import {
...
+import { IonCard, IonHeader, IonCardHeader, IonCardContent, IonCol, IonRow } from '@ionic/react';
```
Comment (grgur) [inline src/Tablet/Sensors/SensorsTab.js]: IonHeader not used

File: src/Tablet/Sensors/SensorsTab.js [hunk 6]
```diff
@@ -96,28 +97,28 @@ const SensorsTab = () => {
 
   const renderCard = type => {
     return (
-      <Col sm={4} style={{ marginTop: 20 }}>
-        <Card>
-          <Card.Header>{type.toUpperCase()}</Card.Header>
-          <Card.Body>{renderType(type)}</Card.Body>
-        </Card>
-      </Col>
+      <IonCol sm={4} style={{ marginTop: 20 }}>
```
Comment (grgur) [inline src/Tablet/Sensors/SensorsTab.js]: `IonCol` doesn't have `sm` property
https://ionicframework.com/docs/api/col

Comment (andrewtran30) [reply src/Tablet/Sensors/SensorsTab.js]: So should the Col stay as Col instead of IonCol?

Comment (grgur) [reply src/Tablet/Sensors/SensorsTab.js]: you can just remove the `sm` prop

File: src/Phone/Sensors/SensorsTab.js (hunk 3 above)
Comment (andrewtran30) [reply src/Phone/Sensors/SensorsTab.js]: which one?

**Pull request merged**